### PR TITLE
Cleaned up setup instructions

### PR DESCRIPTION
**Pull request by britzl**
Made it clearer for an external developer how to set up and run extender locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,103 @@
...
+   * NDK and SDK: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L259-L260
+* iOS and macOS
+   * iOS, macOS, Xcode - previous version: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L475-L485
+   * iOS, macOS, Xcode - latest version: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L488-L496
 
-### Build
+We have prepared scripts to package the required files. Use the scripts in the [defold/scripts/package](https://github.com/defold/defold/tree/dev/scripts/package) folder to create these packages.
 
-* Build the Extender Docker image by running:
+NOTE: If you only plan to use the extender server to build for a single platform you may remove the setup steps for the other platforms to speed up the build process.
```
Comment (JCash) [inline README.md:37]: Not only speed up? If the any platforms are missing, the Dockerfile won't work?

File: README.md [hunk 2]
```diff
@@ -1,90 +1,103 @@
...
-  1. Copy the client to Bob
+```
+$ sudo mkdir /usr/local/extender
+$ sudo chown -R mawe:staff /usr/local/extender
+```
 
-    $ cp -v ./build/libs/extender-client-0.0.1.jar <defold>/com.dynamo.cr/com.dynamo.cr.common/ext/extender-client-0.0.1.jar
+Now the current user has access to the folder and can start the service.
 
+        $ TARGET_DIR=/path/to/localextender ./server/scripts/run-standalone-local.sh
```
Comment (JCash) [inline README.md:98]: What are the prerequisites here in terms of package availability?

File: README_SETUP_RELEASE.md [hunk 3]
```diff
@@ -7,6 +7,15 @@ The following tools need to be installed before [releasing to AWS](README_RELEAS
 * Configure SSH over AWS Session Manager
 
 
+## Submodule setup
+Make sure you have initialized and fetched submodules:
+
+```
+git submodule init
+git submodule update
+```
```
Comment (JCash) [inline README_SETUP_RELEASE.md:16]: I guess this step is optional for some (external) users?

**Review by JCash — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (britzl) [reply README.md:37]: What I meant was not to ignore those steps but to remove them. For instance removing everything related to Emscripten (https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L73-L110) from the Docketfile would mean that Emscripten isn't included in the image and you won't be able to build anything for HTML5 right?

File: README.md (hunk 2 above)
Comment (britzl) [reply README.md:98]: Good point! It should be what we have in the Dockerfile right?

ENV XCODE_13_VERSION        13.2.1
ENV MACOS_12_VERSION        12.1
ENV IOS_15_VERSION          15.2
ENV XCODE_13_CLANG_VERSION  13.0.0
ENV SWIFT_5_5_VERSION       5.5

**Pull request merged**